### ISSUE TITLE
Sveltekit compatibility

### ISSUE THREAD

**Issue by mutil** [jprochazk/svelte-material-icons#1]
Hello and thank you for this nice package.  

When using it in svelte-kit, it prints the following:

> @jprochazk/svelte-material-icons doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.

**Comment by jprochazk** [comment]
Hi,

I personally use this package in a SvelteKit app as well (available [here](https://github.com/ezclap-tv/shit-chat-says/tree/main/scs-ui)). I remember running into this problem, but unfortunately I do not remember the fix 😄.

Please ensure you are importing any icons like so:
```html
<script>
import IconName from "@jprochazk/svelte-material-icons/IconName.svelte"
</script>
```
Where `@jprochazk/svelte-material-icons` is replaced with the `material-icons` alias, if you are using it. Each icon has to be imported separately.

If that isn't helpful, you can also try downgrading to version `0.0.7`.

If none of that works, please share your SvelteKit configuration file.

**Comment by mutil** [comment]
This warning seems to be emitted even without importing the package but by just installing it.
It's only happening in v0.0.8, v0.0.7 does not produce this warning.

Steps to reproduce:
```
npm init svelte@next test1
cd test1
npm i -D @jprochazk/svelte-material-icons
npm run dev
```

Otherwise, it works fine when importing and using the icons.

**Comment by jprochazk** [comment]
Okay, I'll push a new version that fixes it soon.

**Comment by jprochazk** [comment]
Fixed in `0.1.0`.
